### ISSUE TITLE
Do I need to use $$MATH for chord inversions?

### ISSUE THREAD

**Issue by m-k94** [DCMLab/ddd#60]
![image](https://user-images.githubusercontent.com/75686579/112981716-de25c200-915b-11eb-951c-7843ec3ca087.png)
(Weitzmann 1861, p. 12)

I strongly feel like there must be a better way to do this than writing "$$MATH-Accord", but I'm not sure what it is. It's two numbers written above eachother, conventional notation for inversions of chords.

**Comment by fabianmoss** [comment]
Yes I think that's the best way to proceed here. I am a bit surprised that this issue didn't come up earlier but I guess the different sources treat inversions differently.
